uat/testing-features/pom.xml

```diff
@@ -103,6 +103,21 @@
                 <artifactId>maven-resources-plugin</artifactId>
                 <version>2.6</version>
                 <executions>
+                    <execution>
```

Comment:
It would be nice if OTF did this for us and we had a specific structure it could look for (just a thought for the future)